server/webapp/WEB-INF/rails/webpack/views/components/buttons/index.scss

```diff
@@ -81,7 +81,7 @@
 
 .icon-doc {
   @include icon-before($type: $fa-var-file);
-  margin-right: 10px;
+  margin-left: 20px;
```

Comment:
@naveenbhaskar -- I am not sure why we need to add `margin-left` to an icon.
An icon simply should have styles to render an icon. 
The spacing styles of the icon are dependent on the place where the icon is used. I would say for margin and padding, we should wrap the icon in a  different container and style that particular container.

Reply:
This particular component is used in buttons which has an icon with it. Where ever we use this, we need consistent margins and that the only use case this can be used. we dont have any other scenarios where we need to use this. 
wrapping  the buttons in another div just for margin is not a good standard to follow. 
In normal scenarios, we use a second class for this purpose and we don't have that functionality in our components. I would recommend to have that rather than adding an extra div for margin.